### DISCUSSION TITLE
Addition of "hacktoberfest" topic to this repostitory

### DISCUSSION THREAD

**Discussion by kishaningithub** [wailsapp/wails#1992] [Ideas]
In my opinion this repo would attract a lot if new contributors if you can add "hacktoberfest" topic to this repository

More info - https://hacktoberfest.com/participation/#pr-mr-details

**Comment by leaanthony** [comment]
Maybe next year 👍

**Reply by kishaningithub** [reply]
IMO there is still time ☺️
